### ISSUE TITLE
All drag-and-drop functions broken.

### ISSUE THREAD

**Issue by davidsanders** [artf/grapesjs-mjml#95]
When attempting to make any drag-and-drop edits (adding new elements or moving existing elements) results in the [MJML demo](https://grapesjs.com/demo-mjml.html) results in the following console warning:
`Invalid target position: Target collection not found, Target is not droppable, accepts [undefined], Component not draggable, acceptable by [undefined]`

Would love to see a working demo with GrapeJS and MJML!

**Comment by maximilienGilet** [comment]
Same problem here

**Comment by Basit-Mahmood** [comment]
Same Problem here

**Comment by Jonatasmz** [comment]
It's working fine on the latest beta version, 0.15.3

**Comment by davidsanders** [comment]
okay, interesting. the live demo site is running v0.14.52. is there any chance we could get an update to the latest version?

**Comment by Jean-Baptiste-Lasselle** [comment]
Hi all,
I spun up grapesjs-mjml with : 
* `grapesjs` plugin in version `0.14.55`.
* `grapesjs-mjml` plugin in version `0.0.31`.

My results are same as @davidsanders and @Basit-Mahmood  : 
* I do get something close to the oneline demo
* But there are some obvious big differences : for exemple, grammatical / syntaxical / lexical analysis on `MJML`, will not throw any notifications for me, though we do havethat kind of notifications on the live demo
* I also oticed a behavioral difference : the live demo doesn't seem to make the use struggle to drag and drop a 3 columns control, while I do struggle to do the same, on what I deployed. Actually, today, I add columns by editing with atom, and load back into `grapesjs` 

If you'reintereted in my config : 
https://github.com/Jean-Baptiste-Lasselle/grapesjs-mjml-webclient
